Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -390,6 +392,79 @@ void AODProducerWorkflowDPL::addToTracksQATable(TracksQACursorType& tracksQACurs
     trackQAInfoHolder.dTofdZ);
 }
 
+template <typename TRDsExtraCursorType>
+void AODProducerWorkflowDPL::addToTRDsExtra(const o2::globaltracking::RecoContainer& recoData, TRDsExtraCursorType& trdExtraCursor, const GIndex& trkIdx, int trkTableIdx)
+{
+  static int q0s[6] = {-1}, q1s[6] = {-1}, q2s[6] = {-1};
```

Comment:
This is a lot of new detector specific code. Could this not be done in TRD reco for better logic isolation? Also, I would be interested to know the CPU impact on the overall AOD processing.